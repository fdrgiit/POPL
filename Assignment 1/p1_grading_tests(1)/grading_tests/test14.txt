
 @#
 
 $%~IF@WHILE!12345^00?<>

 ^&INT~1234x.11111111111111111112345REAL
 
