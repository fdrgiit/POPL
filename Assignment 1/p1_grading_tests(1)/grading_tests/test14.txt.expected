2 38 ERROR
2 38 ERROR
4 38 ERROR
4 38 ERROR
4 38 ERROR
4 5 IF
4 38 ERROR
4 6 WHILE
4 38 ERROR
4 34 12345 
4 38 ERROR
4 34 0 
4 34 0 
4 38 ERROR
4 26 <>
6 38 ERROR
6 38 ERROR
6 10 INT
6 38 ERROR
6 34 1234 
6 36 x.11111111111111111112345REAL 
